### PR TITLE
disable oplog when only having expiration rules

### PR DESCRIPTION
**Pull request by welansari**
Issue: BB-430

### CODE REVIEW

File: tests/unit/lifecycle/LifecycleOplogPopulatorUtils.spec.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                                abortMpuRule,
+                                removeDeleteMarkersRule,
+                            ]),
+                        ],
+                    },
+                },
+                expectedReturn: false,
+            },
+            {
+                it: 'should return true when there is at least one transition rules',
```
Comment (francoisferrand) [inline tests/unit/lifecycle/LifecycleOplogPopulatorUtils.spec.js]: maybe duplicate this test, with only `nonCurrentTransitionRule` enabled: at the moment i think your test set would work as well if we forgot the NonCurrentTransition in the test...

**Review by francoisferrand — APPROVED** [review]

**Review by rachedbenmustapha — APPROVED** [review]

**Pull request merged**